schemas/context/webinfo.example.1.json

```diff
@@ -0,0 +1,19 @@
+{
+  "xdm:webPage": {
+    "xdm:siteSection": "Product section",
+    "xdm:server": "example.com",
+    "xdm:name": "product home",
+    "xdm:URL": "https://www.example.com/products",
+    "xdm:isErrorPage": false,
+    "xdm:isHomePage": true,
+    "xdm:referrer": {
+      "xdm:URL": "https://www.some-adserver.com",
+      "xdm:referrerType": "external"
+    }
+  },
+  "xdm:webLink": {
+    "xdm:type": "other",
+    "xdm:URL": "https://www.example.com/products/store/?view=1",
```

Comment:
@kstreeter I recommend that we change this example to clearly state the link nature of an intra page action.
"xdm:URL": "https://www.example.com/products?view=stores",
"xdm:name": "product stores expanded"

This will better show the SPA nature of the link event.

Reply:
@cdegroot-adobe  There's no harm in adding `webinfo.example.2.json`